schemas/protocol.fbs

```diff
@@ -4,7 +4,7 @@
 namespace Protocol;
 
 // List of available commands
-union Command (required) { Version, Verack, GetPeers, Peers, Ping, Pong, Block }
+union Command (required) { Version, Verack, GetBlocks, GetPeers, Peers, Ping, Pong, Block }
```

Comment:
It is really important to keep the order in Flatbuffers. That's why we could add new commands into the schema, re-generate the code and everything should work and be compatible.

If the order is changed, then the serialized FlatBuffers will have different internal field IDs.
As a consequence, we will have to update all the tests that use bytes to represent Flatbuffers (what you had to do with the `serializers.rs` tests).